v1/summary.yaml

```diff
@@ -113,9 +113,7 @@ paths:
                 thumbnail: '{{httpsSource(extract.body.items[0].thumbnail)}}'
                 lang: '{{extract.body.items[0].pagelanguagehtmlcode}}'
                 dir: '{{extract.body.items[0].pagelanguagedir}}'
-                # Need to figure out the format of this before we can add it.
-                # See https://phabricator.wikimedia.org/T117082#1787617.
-                #timestamp: '{{request.params.timestamp}}'
+                timestamp: '{{extract.body.items[0].touched}}'
```

Comment:
In https://en.wikipedia.org/w/api.php?action=query&format=json&prop=extracts|pageimages|revisions&formatversion=2&redirects=true&exintro=true&exsentences=5&explaintext=true&piprop=thumbnail&pithumbsize=600&rvprop=timestamp&titles=Master_System&smaxage=300&maxage=300&uselang=content the revision ("last edited" timestamp seems to be `body.items[0].revisions[0].timestamp`. Normally, `touched` is updated whenever the page needs to be re-rendered, which might not be the "last edited at" semantics we are looking for here.